### DISCUSSION TITLE
Mobility Working Group Call 9 - Dec 2nd 2022

### DISCUSSION THREAD

**Discussion by bharath-ganapathy** [beckn/mobility#30] [General]
Attendees:
Ravi Prakash
Bharath Ganapathy
Shilpi Mohan
Priyanka Yadav
Nitin Nair
Venky Mahadevan
Ashish Baweja
Rupal Singla
Rajesh Rajagopalan
Anitha Rajagopalan
Romel Kumar
Hariharan Thiyagarajan

Presentation slides:https://docs.google.com/presentation/d/12NuBaIfLB5AuUtgz-Ci7R1jRA8GTqnzuEf-Jrj6FyYo/edit?usp=share_link

Meeting recording:
https://drive.google.com/file/d/1cW9L6yduO3w_WpmRmH9-lHQQKVln2us2/view?usp=share_link

Confluence link for documents:
https://fide-official.atlassian.net/wiki/spaces/MO/pages/2228280/Use+Case+Description+Template

**Comment by ShilpiMKumar** [comment]
Action Items as discussed on Dec 2nd  : 

1. Please pick a use case category from discussion and document happy flow considering all stages of the order lifecycle.
2. Illustrate the use case flows as sequence diagrams or flow diagrams
